src/client/Microsoft.Identity.Client/WsTrust/WsTrustEndpoint.cs

```diff
@@ -80,7 +80,7 @@ private string BuildTokenRequestMessage(UserAuthType authType, string cloudAudie
                     writer.WriteString(soapAction);
                     writer.WriteEndElement(); // Action
 
-                    writer.WriteStartElement("messageID", wsaNamespaceValue);
+                    writer.WriteStartElement("MessageID", wsaNamespaceValue);
```

Comment:
why?

Reply:
A user pointed this out in the issue. we were not following the spec
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2152#issuecomment-722491054